plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -153,6 +154,17 @@ get_product_name (GError **error)
 	return g_strndup (image_version, hyphen_index - image_version);
 }
 
+static FlatpakInstallation *
+get_user_installation (GError **error)
```

Comment:
`get_user_installation()` is a very thin wrapper around `flatpak_installation_new_user()` which doesn’t actually do any computation of its own. Why does it exist?